container-core/src/main/java/com/yahoo/container/jdisc/ThreadedHttpRequestHandler.java

```diff
@@ -31,6 +32,7 @@
 
     public static final String CONTENT_TYPE = "Content-Type";
     private static final String RENDERING_ERRORS = "rendering_errors";
+    private static final String UNHANDLED_EXCEPTION_METRIC = "jdisc.http.handler.unhandled_exception";
```

Comment:
Shouldn't it be `..unhandled_exceptions` with the plural 's'? If so, also rename the constant.

Reply:
Agree - plural sounds better. I will update the PR with a rename.

Reply:
👍